packages/config/config/devices/0x0330/zv9001k12-dim-z4.json

```diff
@@ -1,21 +1,54 @@
-// ShenZhen Sunricher Technology, Ltd. SR-ZV9001K12-DIM-Z4
-// Z-Wave Dim Remote Control
+// Sunricher ZV9001K12-DIM-Z4
+// 4 Channel Dimmer Remote Control
 {
-	"manufacturer": "ShenZhen Sunricher Technology, Ltd.",
+	"manufacturer": "Sunricher",
 	"manufacturerId": "0x0330",
-	"label": "SR-ZV9001K12-DIM-Z4",
-	"description": "Z-Wave Dim Remote Control",
+	"label": "ZV9001K12-DIM-Z4",
+	"description": "4 Channel Dimmer Remote Control",
 	"devices": [
 		{
 			"productType": "0x0300",
 			"productId": "0xb302",
 			"zwaveAllianceId": 3907
+		},
+		{
+			"productType": "0x0301",
+			"productId": "0xb302",
+			"zwaveAllianceId": 3907
+		},
+		{
+			"productType": "0x0302",
+			"productId": "0xb302",
+			"zwaveAllianceId": 3907
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Launch 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		}
+	},
```

Comment:
These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.